guides/messaging-high-availability.adoc

```diff
@@ -0,0 +1,227 @@
+= Configure WildFly With Messaging (ActiveMQ Artemis) Cluster And High Availability
+:summary: Learn how to configure WildFly with messaging (ActiveMQ Artemis) cluster and high availability
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+// you can override any attributes eg to lengthen the
+// time to complete the guide
+:prerequisites-time: 20
+
+In this guide you will learn how to configure two WildFly servers with messaging (integrated ActiveMQ Artemis broker)
+in high availability topology with https://activemq.apache.org/components/artemis/documentation/latest/ha.html#shared-store[shared journal].
```

Comment:
s/in high/in a high/g
s/with shared journal/with a shared journal/g

Reply:
also, use...

[shared journal, window=_blank]

here and in other links, at least all those that appear in the main content.

That opens the link in a new tab, leaving the existing tab on the guide page.